src/test/test.spec.js

```diff
@@ -0,0 +1,235 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import { describe, it } from 'mocha';
+import app from '../app';
+
+chai.use(chaiHttp);
+
+describe('SendIT API TESTS', () => {
+  // Test Invalid pathss
+  describe('/GET INVALID_PATH', () => {
+    it('should return not found', () => chai.request(app)
+      .get('/INVALID_PATH')
+      .then((res) => {
+        expect(res).to.have.status(404);
+      }));
+  });
+
+  // Test Parcels' requests
+  describe('Parcels', () => {
+    // Create parcel
+    describe('POST/ Parcel', () => {
+      // Parcel should be created
+      it('should create a parcel', () => {
+        const parcel = {
+          userId: 1,
+          parcel: {
```

Comment:
This can be put under Mocha's `beforeTest` method. Kindly refer to this link so that you learn how to create mock test data with Mocha. 
*  https://stackoverflow.com/questions/32660241/mocha-beforeeach-vs-before-execution
* https://mochajs.org/